docusaurus.config.js

```diff
@@ -42,10 +42,28 @@ const config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       ({
         docs: {
-          sidebarPath: require.resolve('./sidebars.js'),
-          // Please change this to your repo.
-          // Remove this to remove the "edit this page" links.
+          // Basic Docs Preferences
+          showLastUpdateAuthor: true,
+          showLastUpdateTime: true,
+          editCurrentVersion: true,
+          editLocalizedFiles: true,
           editUrl: 'https://github.com/subspace/subspace-docs/blob/main/',
+          sidebarPath: require.resolve('./sidebars.js'),
+
+          //Versioning Preferences
+          lastVersion: "latest",
+          versions: {
+            latest: {
+              label: 'Latest',
+              path: '',
+            },
+            current: {
```

Comment:
I'd suggest keeping the "Latest" as a default version and provide switching to "Pre-release" as an option. 
I'm afraid some users might not pay attention to the selected version and accidentally install the pre-release version, which could be unstable.

Reply:
The default is defined by the `lastVersion` value on line 54, it is set to be latest. 

It is worth noting that docusaurus has good cookie and preference memory, it will remember what version you selected and keep you on that version as you navigate or reload the site to overwrite this value, but by default people should get `latest`